src/screens/expandedAssetSheet/components/sections/AboutSection.tsx

```diff
@@ -210,7 +210,7 @@ export function AboutSection() {
 
   return (
     <Box gap={40}>
-      <Box gap={4} marginBottom={rowItems.length % 2 === 0 ? '-12px' : undefined}>
+      <Box gap={4} marginBottom={rowItems?.length % 2 === 0 ? '-12px' : undefined}>
```

Comment:
A lot of this optional chaining seems a bit random. Also generally would prefer if we corrected incorrect types for any truthy types that genuinely can be falsy, vs. creating misalignment between types and logic.